practice-one/src/components/CustomerAvatar/CustomerAvatar.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import "./CustomerAvatar.css";
+
+const CustomerAvatar = ({ customerAvatar, customerName }) => {
```

Comment:
rename CustomerInfo instead of CustomerAvatar

Reply:
Yes, I renamed all files and component
![image](https://github.com/datdragon777/React-Training/assets/74806826/e551ef7e-6f56-439b-945e-4af5ccd87e3e)
![image](https://github.com/datdragon777/React-Training/assets/74806826/b71416d5-e6bf-4503-baf4-c4df2335fd8b)